sys/include/phydat.h

```diff
@@ -98,6 +98,7 @@ enum {
     /* pressure */
     UNIT_BAR,       /**< Beer? */
     UNIT_PA,        /**< Pascal */
+    UNIT_HPA,       /**< hectoPascal */
```

Comment:
okay, I remove `hPa` from this PR. Still, might be worth to further think about improving SAUL to (maybe) allow separate scaling of units and values.